### ISSUE TITLE
SSL Errors

### ISSUE THREAD

**Issue by qwksilver** [intiface/intiface-tutorial#7]
water fox 56.2.10.1
Secure Connection Failed, An error occurred during a connection to localhost:12345. SSL received a record that exceeded the maximum permissible length. Error code: SSL_ERROR_RX_RECORD_TOO_LONG

chrome 74.0.3729.169 (Official Build) (64-bit)
This site can’t provide a secure connection localhost sent an invalid response.
Try running Windows Network Diagnostics. ERR_SSL_PROTOCOL_ERROR

errors come from clicking "Click here to open a new tab and accept the certificate."

also "when trying to connect again"
The connection failed. If you look at the server dialog, you may see something like this:
This means we need to set up the self-signed security certificate in your browser.
To do this, we'll need to visit the websocket server, then accept the certificate.

**Comment by qdot** [comment]
Is this from trying to connect out to Intiface Desktop?

**Comment by qwksilver** [comment]
yes, i got it to connect by finding hidden somewhere in the docs to switch the wss:// to ws://  which fixed the problem, but all the instructions to add the SSL Certificate earned me the above errors on those platforms and the wss:// wouldn't connect at all.

**Comment by qdot** [comment]
Yup, this is a UI problem that I'm hoping to get fixed ASAP. Thanks for pointing it out, and I'll leave this open 'til it's fixed.

**Comment by anoadragon453** [comment]
Not only this, but intiface desktop by default only accepts connections from `127.0.0.1`. The tutorial tries to use `localhost` which the server apparently doesn't accept.

Either the desktop app should accept both, or the tutorial should use `127.0.0.1` in its URL.

I found this on Debian, using the AppImage btw, which might possibly have stricter rules than OS'.

**Comment by qwksilver** [comment]
I remember what it would take to replicate, but i'm also sure that 0.5.0 fixed it.  I installed the update to the cli engine on the intiface-desktop with the server running.  When it ran the installer for the update, it didn't successfully close the server app, so likely a key file was locked open.  I did the exact same thing with the last update to 0.5.0 and thought wow this is stupid, right after I hit ok/go and didn't recreate the error.  i think this issue might be closed.

**Comment by qdot** [comment]
@qwksilver Oh! Actually, this isn't completely fixed yet here, and it's actually an intiface desktop error. The Updater shouldn't be allowed to run when the server is open. I've filed https://github.com/intiface/intiface-desktop/issues/65 to fix that, will close this.

@anoadragon453 I've filed your bug over in intiface-cli-node for right now, as that's a problem in the engine (which is where the ws server sets up). 

Closing this bug out now since we've got it divided into other areas.
